src/apps/reservation-list/modal/reservation-details/reservation-details-redirect.tsx

```diff
@@ -29,6 +29,7 @@ const ReservationDetailsRedirect: FC<
     <div className={`modal-details__buttons ${className}`}>
       <button
         type="button"
+        data-cy="delete-reservation"
```

Comment:
`dataCy` is used in [reservation-details-buttons.tsx](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/425/files#diff-178291d6591a1bad453329a81156a69812e7e3acb9a685b2fa46a9a8a15656e3)
should `data-cy` be the same?